app/config/routing.yml

```diff
@@ -1,3 +1,11 @@
+demo:
+    resource: "@DemoBundle/Controller/"
+    type:     annotation
+    prefix:   /demo
+
 app:
     resource: "@AppBundle/Controller"
     type: annotation
+
+fos_user:
+    resource: "@FOSUserBundle/Resources/config/routing/all.xml"
```

Comment:
No mew line at the end of file, always add a single new line at end of each file This will help git to merge changes in some cases